src/api/boardReaction.ts

```diff
@@ -0,0 +1,24 @@
+import {SERVER_HTTP_URL} from "config";
+import {BoardReactionType, ReactionType} from "types/reaction";
+
+export const BoardReactionAPI = {
+  addBoardReaction: async (boardId: string, reactionType: ReactionType) => {
```

Comment:
I don't know if this is the right location for this, and also it's just a general idea, but: Should we add some kind of limit/debounce to sending board reactions? Right now you can spam them to the point where the application starts to lag, and that's just from one person doing it.

Reply:
I tried to implement a hook that can debounce multiple things so we can keep track of each reaction and maybe even have a small visual timer (like the background of the emoji changing progress circle), but something didn't work so I'll have to take another look ^^

Reply:
I feel like a little debounce hook (we could just use a library for that) on the reactions in general would be enough, so that one user can only send a limited amount of reactions per second would be enough, without any visual timer. I think this is how Google Meet does it.
Imo, for everything else we could create a new issue. What do you think?

Reply:
👍 agreed

Reply:
debounce is now implemented